src/Core/src/ImageSources/UriImageSourceService/UriImageSourceService.Android.cs

```diff
@@ -19,7 +19,10 @@ public partial class UriImageSourceService
 				{
 					var callback = new ImageLoaderCallback();
 
-					PlatformInterop.LoadImageFromUri(imageView, uriImageSource.Uri.OriginalString, new Java.Lang.Boolean(uriImageSource.CachingEnabled), callback);
+					var cachingEnabled = uriImageSource.CachingEnabled
+						? Java.Lang.Boolean.True
+						: Java.Lang.Boolean.False;
+					PlatformInterop.LoadImageFromUri(imageView, uriImageSource.Uri.OriginalString, cachingEnabled, callback);
```

Comment:
@jonathanpeppers  the `new Java.Lang.Boolean()` ctor is obsolete, so I am using the field. Is that correct?